packages/lib-classifier/src/components/Classifier/components/Banners/components/AlreadySeenBanner/AlreadySeenBannerContainer.stories.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import styled from 'styled-components'
+import { AlreadySeenBannerContainer } from './AlreadySeenBannerContainer'
+import readme from '../../README.md'
+import backgrounds from '../../../../../../../.storybook/lib/backgrounds'
```

Comment:
If these are common to all stories, could they be moved to the storybook config? There is an ability to add global decorators: https://storybook.js.org/docs/addons/introduction/#storybook-decorators

Reply:
Which background is the default varies for the story, so I don't think the global decorator will work in this case. Happy to be wrong though.